app/assets/javascripts/datatables.coffee

```diff
@@ -0,0 +1,24 @@
+$(document).on 'turbolinks:load', ->
+  tableElementIds = [
+    '#passengers.admin-table'
+    '#passengers.dispatch-table'
+  ]
+  i = 0
+  while i < tableElementIds.length
+    tableElementId = tableElementIds[i]
+    if $.isEmptyObject($.find(tableElementId))
```

Comment:
Aren't these IDs _always_ on the page? any reason to expect `$.find` not to find them?